frontend/public/plugins.ts

```diff
@@ -30,13 +26,30 @@ export const initConsolePlugins = _.once((reduxStore: Store<RootState>) => {
   registerPluginEntryCallback(pluginStore);
 });
 
+const loadPluginFromURL = async (baseURL: string) => {
+  const manifest = await fetchPluginManifest(baseURL);
+  loadDynamicPlugin(baseURL, manifest);
+  return getPluginID(manifest);
+};
+
+// Load all dynamic plugins which are currently enabled on the cluster
+window.SERVER_FLAGS.consolePlugins.forEach((pluginName) => {
+  loadPluginFromURL(`${window.SERVER_FLAGS.basePath}api/plugins/${pluginName}`).then((pluginID) => {
+    pluginStore.setDynamicPluginEnabled(pluginID, true);
+  });
```

Comment:
I assume we should have a `catch` here with error handling? Maybe just a `console.error`, and we can follow up later and show something in the UI if a plugin doesn't load.

Reply:
Good idea, I'll add a `catch` handler.

Loading a dynamic plugin is essentially a two-step async operation:

```ts
// 1. async, need to await the result
const manifest = await fetchPluginManifest(baseURL);

// 2. also async, but without a result, instead we expect a side effect
// (plugin entry script should call window.loadPluginEntry global function)
loadDynamicPlugin(baseURL, manifest);
```

`fetchPluginManifest` already logs any validation error details to browser console.

`loadDynamicPlugin` and the `window.loadPluginEntry` callback already report any errors to browser console as well.

But having a `catch` handler is still useful since it can report on issues like `coFetch` failing or other async operations which don't log error details.